pkgs/development/libraries/mesa/default.nix

```diff
@@ -9,7 +9,7 @@
 , galliumDrivers ? ["auto"]
 , vulkanDrivers ? ["auto"]
 , eglPlatforms ? [ "x11" ] ++ lib.optionals stdenv.isLinux [ "wayland" ]
-, vulkanLayers ? [ "device-select" "overlay" ]
+, vulkanLayers ? lib.optionals stdenv.isLinux [ "device-select" "overlay" ]
```

Comment:
Could you explain what this does and why it fixes the darwin build (and ideally add that to the commit message)?

Is this really linux-specific or just a not-darwin thing?

Reply:
```suggestion
, vulkanLayers ? lib.optionals !stdenv.isDarwin [ "device-select" "overlay" ] # No Vulkan support on Darwin
```

Reply:
Done.